docs/metrics.md

```diff
@@ -0,0 +1,171 @@
+# METRICS
+
+## Data Analysis
```

Comment:
From a high level, what are we trying to accomplish with this experiment? E.g. is this meant to justify a general launch for the feature or are we looking specifically at whether the UI is prominent enough?